src/engraving/dom/score.cpp

```diff
@@ -4395,7 +4395,7 @@ void Score::removeSpanner(Spanner* s)
 bool Score::isSpannerStartEnd(const Fraction& tick, track_idx_t track) const
 {
     for (auto i : m_spanner.map()) {
-        if (i.second->track() != track) {
+        if (i.second->track() != track && !i.second->isGradualTempoChange()) {
```

Comment:
My expectation was to check the "isSystem()" property, but this is *not* true for gradual tempo changes.  But gradual tempo changes can end up in a different staff to the segments that are selected when you add them - they appear to be only type of element for which this is true.